boards/arm/nrf52833dk_nrf52833_mdbt/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NRF52833DK_NRF52833_MDBT
```

Comment:
If it is about MDBT50Q-DB-33 board then it should be named MDBT50Q-DB-33, `config BOARD_MDBT50Q_DB_33`